rfcs/text/0000-matrix-service.md

```diff
@@ -0,0 +1,413 @@
+"filter": {
+  "op": ">=",
+  "value": "full length",
+  "field": "library_preparation_protocol.end_bias"
+}
+```
+
+Ex. Select all cells from the "Single cell transcriptome analysis of human pancreas" project with at least 3000 genes detected:
+
+```
+"filter": {
+  "op": "and",
+  "value": [
+    {
+      "op": "=",
+      "value": "Single cell transcriptome analysis of human pancreas",
+      "field": "project.project_core.project_short_name"
+    },
+    {
+      "op": ">=",
+      "value": 3000,
+      "field": "genes_detected"
+    }
+  ]
+}
+```
+
+#### Fields
+
+Users may supply a list of metadata `field`s to be exported with their generated matrix. For `loom` formats, metadata is made available within the `loom` file itself via an adjacent attribute matrix; `mtx` and `csv` formats provide a separate CSV file to serve user requested metadata. The full list of available metadata fields is available at the [/fields endpoint](https://matrix.data.humancellatlas.org/v1/fields). More information about a specific metadata field such as a description and a set of valid values is available at the `/fields/<field>` endpoint. See [Auxiliary Endpoints](#auxiliary-endpoints) for more details on these.
+
+#### Format
+
+The API generates expression matrices in three formats: `loom`, `mtx` and `csv`. From the [loom](http://loompy.org/) documentation page, the format is "an efficient file format for large omics datasets", enabling the storage of large expression matrices with metadata and also supports many popular data processing programming languages. For these reasons, `loom` is the service's default and most performant format. The `mtx` format adheres to the [10x feature-barcode matrix specification](https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/output/matrices) to enable compatibility with 10X methods from `scanpy` and `Seurat`. `csv` is supported as a raw text format for which performance may not be optimized.
+
+#### Feature
+
+Users are also able to specify the expression feature type, i.e. the matrix's axis label, as `gene` (default) or `transcript` names.
+
+#### Species (proposed)
+
+`species` is a potential fifth parameter that the endpoint may accept. Currently, `species` is not a top-level parameter and is rather specified within `filter` via the `cell_suspension.genus_species.ontology` and `cell_suspension.genus_species.ontology_label` filters. Since a distinct matrix is generated for each `species` requested, it may be a more intuitive user experience for it to be a top-level parameter. Additionally, it requires non-trivial logic for the Matrix Service to parse a complex `filter` object to determine which species are requested, i.e. which species to generate a matrix for. In order to simplify the user's mental model of the API as well as the service's logic complexity, `species` is proposed as a fifth request body parameter.
+
+*Note: Currently, supported species are Homo Sapiens and Mus Musculus.*
```

Comment:
```suggestion
*Note: Currently, supported species are _Homo sapiens_ and _Mus musculus_.*
```

Reply:
The note is already italicized - will only italicize species names.

Reply:
Sorry, didn't realize that. Would've been fine then.
Please lowercase the species names